### PR TITLE
Add RTL support

### PR DESCRIPTION
**Pull request by yuriy-fix**
Connected to #106

### CODE REVIEW

File: theme/lumo/vaadin-tab-styles.html [hunk 1]
```diff
@@ -189,6 +189,40 @@
...
+        transform: translateX(50%) scale(1);
+      }
+
+      :host ::slotted(iron-icon:first-child) {
+        margin-right: 0;
+      }
+
+      :host ::slotted(iron-icon:last-child) {
+        margin-left: 0;
+      }
```
Comment (DiegoCardoso) [inline theme/lumo/vaadin-tab-styles.html]: Should these be scoped to `[dir="rtl"]`?

Comment (yuriy-fix) [reply theme/lumo/vaadin-tab-styles.html]: Good catch! Updated selectors

File: theme/material/vaadin-tabs-styles.html [hunk 2]
```diff
@@ -76,8 +76,7 @@
       /* RTL specific styles */
 
       :host([dir="rtl"]) [part="forward-button"] {
-        left: auto;
-        right: 0;
+        left: 0;
```
Comment (DiegoCardoso) [inline theme/material/vaadin-tabs-styles.html]: Shouldn't it be:
```suggestion
        right: 0;
```

Comment (yuriy-fix) [reply theme/material/vaadin-tabs-styles.html]: Hmmm, the `:host(:not([dir="rtl"])) [part="forward-button"]` is `right: 0;`, so this one should be `left: 0`

Comment (DiegoCardoso) [reply theme/material/vaadin-tabs-styles.html]: Ah, okay. My bad.

**Pull request merged**